ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -822,11 +822,9 @@ MiSnapThunk(IN PVOID DllBase,
             /* Now assume failure in case the forwarder doesn't exist */
             Status = STATUS_DRIVER_ENTRYPOINT_NOT_FOUND;
 
-            /* Build the forwarder name */
+            /* Build the forwarder name, including the dot */
             DllName.Buffer = (PCHAR)Address->u1.Function;
-            DllName.Length = (USHORT)(strchr(DllName.Buffer, '.') -
-                                      DllName.Buffer) +
-                                      sizeof(ANSI_NULL);
+            DllName.Length = (USHORT)(strchr(DllName.Buffer, '.') - DllName.Buffer + strlen("."));
```

Comment:
Wrong.

Reply:
What is wrong?

Reply:
Changing kernel code for the sake of changing it? using strlen here is retarded.